ftest.sh

```diff
@@ -145,6 +145,18 @@ fi
 trap 'set +e; cleanup' EXIT
 
 CLUSH_ARGS=($CLUSH_ARGS)
+if ! clush "${CLUSH_ARGS[@]}" -B -l "${REMOTE_ACCT:-jenkins}" -R ssh -S \
```

Comment:
(lint) Did you forget the 'then' for this 'if'? [SC1049]
(lint) Couldn't parse this if expression. [SC1073]